src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -488,6 +488,11 @@ public void testAllCheckstyleChecksHaveMessage() throws Exception {
         for (Class<?> module : CheckUtil.getCheckstyleChecks()) {
             final String name = module.getSimpleName();
 
+            // hidden exception messages
+            if ("SuppressWarningsHolder".equals(name)) {
```

Comment:
please remove comment, I do not understand it. Exception will fail whole Checkstyle execution, it is not validation, any Check can throw exception. Or make comment `// No messages in just module`

Please try to remove `SuppressWarningsHolder` from `getCheckstyleChecks` return, if there a lot of other changes(please share them here) are required then lets keep it and we will create new issue on this and address separately.

Reply:
@romani  I updated the comment to // No messages in just module

and I am waiting for Travis to rebase if it's failed

in `getCheckstyleChecks` we can add one more filter to remove `SuppressWarningsHolder` from the result 

    public static Set<Class<?>> getCheckstyleChecks() throws IOException {
            final ClassLoader loader = Thread.currentThread().getContextClassLoader();
            final String packageName = "com.puppycrawl.tools.checkstyle";
            return getCheckstyleModulesRecursive(packageName, loader).stream()
                    .filter(ModuleReflectionUtil::isCheckstyleTreeWalkerCheck)
                    .filter(aClass -> "SuppressWarningsHolder".equals(aClass.getName()))
                    .collect(Collectors.toSet());
    }

Reply:
Ok, lets do this in separate PR , new issue https://github.com/checkstyle/checkstyle/issues/8121 .
You are welcome to fix it as you know all details.